_includes/user-menu.html

```diff
@@ -42,6 +42,7 @@
 		<ul>
             <li><a href="{{docu}}/features/index.html">Overview</a></li>
             <li><a href="{{docu}}/features/sitemap.html">Sitemaps</a></li>
+            <li><a href="{{docu}}//features/things-items.html">Things and Items</a></li>
             <li><a href="{{docu}}/features/persistence.html">Persistence</a></li>
```

Comment:
OK, all done. I've moved a few menus, and added them based on the link. Further review of the Rules and scripting section will need to be made by the powers that be.